client/blocks/importer/wordpress/import-everything/pre-migration/upgrade-plan.tsx

```diff
@@ -49,7 +49,7 @@ export const PreMigrationUpgradePlan: React.FunctionComponent< Props > = ( props
 		dispatch(
 			recordTracksEvent( 'calypso_site_migration_upgrade_plan_screen', migrationTrackingProps )
 		);
-	}, [] );
+	}, [ dispatch, migrationTrackingProps ] );
```

Comment:
`calypso_site_migration_upgrade_plan_screen` gets triggered again when clicking on `Upgrade and migrate`